coltypes/coltypes.go

```diff
@@ -52,24 +54,30 @@ func TransformRow(row map[string]any, columnTypes map[string]ColumnData) (map[st
 			continue
 		}
 
-		switch data := columnTypes[key]; {
-		case data.Type == numberType:
+		data := columnTypes[key]
+		if data.Type == numberType {
```

Comment:
maybe change data.Type to columnTypes[key].Type

Reply:
Done.